torchmetrics/functional/regression/r2.py

```diff
@@ -21,6 +21,15 @@
 
 
 def _r2_score_update(preds: Tensor, target: Tensor) -> Tuple[Tensor, Tensor, Tensor, Tensor]:
```

Comment:
I'm a bit skeptical about the variable names defined in this function. For example, `sum_error = torch.sum(target, dim=0)`. Hence the docstring for `_r2_score_compute` is currently incomplete.

Reply:
hi, how would you suggest to change it?

Reply:
Currently it is as follows,
```
sum_error = torch.sum(target, dim=0)
sum_squared_error = torch.sum(target * target, dim=0)
diff = target - preds
residual = torch.sum(diff * diff, dim=0)
total = target.size(0)
```
Suggested,
```
sum_obs = torch.sum(target, dim=0)
sum_squared_obs = torch.sum(target * target, dim=0)
residual = target - preds
rss = torch.sum(residual * residual, dim=0)
n_obs = target.size(0)
```
And similar changes in the argument names for `_r2_score_compute()`

Reply:
yep, that sounds good to me :rabbit:

Reply:
I have made the required changes.